site-src/guides/implementers.md

```diff
@@ -27,6 +27,13 @@ like APIs delivered in the core Kubernetes binaries, each time a release happens
 the APIs have been reviewed by upstream Kubernetes reviewers, just like the APIs
 delivered in the core binaries.
 
+### API Semantics
+
+Gateway API MUST follow the Kubernetes API semantics. 
+The concepts are documented [here](https://kubernetes.io/docs/reference/using-api/api-concepts/).
```

Comment:
I'm not sure how these are related to Server-Side Apply?

Reply:
So I expect the client changes to be used by consumers of the Gateway API. Eg. two users managing different listeners on a single Gateway.


For implementors you get SSA for free from the k8s API. So I thought you wanted me to highlight what SSA is for those who aren't using the k8s API server for their control plane. A precursor to that is know the k8s concepts. 

I guess I'm not sure what you're looking for exactly. So let me know.

Reply:
OK, I guess I'm just confused then. I thought the purpose of generating all this new code was so that implementations could use it. What is all the new generated code doing?

Reply:
SSA requires separate go types (apply configurations) and codegen adds Apply methods to existing client sets 

eg. existing k8s APIs have their apply configurations here 

https://github.com/kubernetes/client-go/tree/master/applyconfigurations


The new types make all fields optional so the server can infer what's set from what's a zero value